spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisionerTests.java

```diff
@@ -141,49 +141,10 @@ void testProvisionConsumerDestination_customSubscription() {
     assertThat(result.getName()).isEqualTo("my-custom-subscription");
   }
 
-  @Test
-  void testProvisionConsumerDestination_noTopicException() {
-    when(this.pubSubConsumerProperties.isAutoCreateResources()).thenReturn(false);
-    when(this.pubSubAdminMock.getTopic("topic_A")).thenReturn(null);
-
-    assertThatExceptionOfType(ProvisioningException.class)
-        .isThrownBy(
-            () ->
-                this.pubSubChannelProvisioner.provisionConsumerDestination(
-                    "topic_A", "group_A", this.properties))
-        .withMessage("Non-existing 'topic_A' topic.");
-  }
-
-  @Test
-  void testProvisionConsumerDestination_noSubscriptionException() {
-    when(this.pubSubConsumerProperties.isAutoCreateResources()).thenReturn(false);
-
-    assertThatExceptionOfType(ProvisioningException.class)
-        .isThrownBy(
-            () ->
-                this.pubSubChannelProvisioner.provisionConsumerDestination(
-                    "topic_A", "group_A", this.properties))
-        .withMessage("Non-existing 'topic_A.group_A' subscription.");
-  }
```

Comment:
Previously:
> When auto-create-resources is `false`, and `topicName.groupName` does not result in an existing subscription, then throw a ProvisioningException.

Now:
> When auto-create-resources is `false`, and `topicName.groupName` does not result in an existing subscription, then return a new PubSubConsumerDestination instance.

Is this change intentional? Could you clarify?

Reply:
Yes, as a Pub/Sub Subscriber role `consumer` does not have access to `getSubscription` and it is not required as well until resource needs to be created by specifying `isAutoCreateResources=true`. 
So this test is no longer required  as it will not check for subscription and in case `isAutoCreateResources` is true it will create a subscription if not available.

Reply:
As I understand, this no longer validates the existence of the subscription, thereby allowing the more limited-access "Pub/Sub Subscriber" role to work.